doc/qos_overview.md

```diff
@@ -422,3 +429,176 @@ markings. The same interface has an egress scheduler policy applied to it.
   }
 }
 ```
+
+### Ingress Classification with Ingress Scheduling (Policer)
+
+The example QoS configuration below shows the configuration of an interface,
+assumed to be facing a customer which has ingress classification based on
+the next-hop group the packet is being sent to.  The same interface has an
+ingress scheduler policy applied to it which implementes a ONE_RATE_TWO_COLOR
+policer.
+
+In this scenario, the device is does not have hardware or software to implement
+in ingress queue.  To satisfy the OC schema requirements, a dummy or "fake"
+queue is created.
+
+```yaml
```

Comment:
Given that we don't have a spec for YANG modelled data being serialised to YAML, can we write this as JSON please?

Reply:
Converted to legal JSON, so the comments are gone.    Do you want me to add them back in even though it cannot be validated?  😜